### DISCUSSION TITLE
Add additional sensors (NetPrice)

### DISCUSSION THREAD

**Discussion by itsh-neumeier** [mampfes/ha_epex_spot#93] [Ideas]
Is it possibile to add NetPrice sensor as Value in ct/kWh and one as Euro/kWh

**Comment by mampfes** [comment]
Sorry, I don't want to add more sensors just for this quite useless currency conversion.
What would you think of the following proposal:
The unit-of-measure can be configured (in ct/kWh, EUR/kWh, EUR/MWh, ...) and is valid for all sensors (price, net price, avg, etc.).
